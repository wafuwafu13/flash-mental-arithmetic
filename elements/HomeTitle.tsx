import React from 'react';
import { StyleSheet, Text } from 'react-native';

const HomeTitle = () => {
    return(
        <Text style={styles.title}>
            多面！フラッシュ暗算
        </Text>
    )
}

export default HomeTitle;

const styles = StyleSheet.create({
    title: {
      color: '#fff',
      fontSize: 24,
      top: 100
    }
});
