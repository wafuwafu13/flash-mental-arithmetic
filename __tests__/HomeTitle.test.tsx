import React from 'react';
import HomeTitle from '../elements/HomeTitle'
import renderer from 'react-test-renderer';


it('renders correctly with defaults', () => {
  const button = renderer
    .create(<HomeTitle />)
    .toJSON();
  expect(button).toMatchSnapshot();
});