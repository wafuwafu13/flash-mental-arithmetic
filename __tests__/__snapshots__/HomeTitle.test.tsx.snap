// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly with defaults 1`] = `
<Text
  data-test="homeTitle"
  style={
    Object {
      "color": "#fff",
      "fontSize": 52.5,
    }
  }
>
  多面！フラッシュ暗算
</Text>
`;
